pkg/network/domainspec/generators.go

```diff
@@ -317,6 +321,37 @@ func (b *PasstLibvirtSpecGenerator) Generate() error {
 	iface := b.generateInterface(&b.domain.Spec.Devices.Interfaces[passtIfaceIdx])
 	b.domain.Spec.Devices.Interfaces[passtIfaceIdx] = *iface
 
+	go func() {
+		for {
+			if _, err := os.Stat(passtLogFile); !errors.Is(err, os.ErrNotExist) {
+				break
+			}
+			time.Sleep(time.Second)
+		}
+
+		logFile, err := os.Open(passtLogFile)
+		if err != nil {
+			log.Log.Reason(err).Errorf("failed to open file: %s", passtLogFile)
+			return
+		}
+		defer func() {
+			if closeErr := logFile.Close(); closeErr != nil {
+				log.Log.Reason(err).Errorf("failed to close file: %s", passtLogFile)
+			}
+		}()
```

Comment:
I think there is a memory leak here. I think only the lask opened fd will be closed.
Please see the example here- https://stackoverflow.com/questions/16010694/how-golangs-defer-capture-closures-parameter

Reply:
The file get opened one time only so there is only on FD to deal with, isn't it?

Reply:
@AlonaKaplan , is this comment still relevant?
I think the intention is to open only one fd, if it succeeds, it breaks the loop and the defer is set to close it.

Reply:
no